tests/components/application-connector/resources/charts/gateway-test/charts/test/templates/applications/credentials/mtls-oauth-nagative-other-ca.yaml

```diff
@@ -6,6 +6,6 @@ metadata:
 type: Opaque
 data:
   {{- $files := .Files }}
-  crt: {{ $files.Get "certs/negative/client.crt" | b64enc  }}
-  key: {{ $files.Get "certs/negative/client.key" | b64enc  }}
-  clientId: {{ "someClientID1" | b64enc }}
+  crt: {{ $files.Get "certs/invalid-ca/client.crt" | b64enc  }}
+  key: {{ $files.Get "certs/invalid-ca/client.key" | b64enc  }}
+  clientId: {{ "clientID" | b64enc }}
```

Comment:
I think in the tests, we can change all the clientIDs from `someClientID...` to just `clientID`, right?

Reply:
From now on, the `clientId` field is not used to create the mTLS cache key. We can set it to any value in the corresponding tests.

Reply:
Yes, we can also change that